front/src/views/Boxes/components/BoxesTable.tsx

```diff
@@ -34,6 +35,7 @@ interface IBoxesTableProps {
   actionButtons: React.ReactNode[];
   columnSelector: React.ReactNode;
   onBoxRowClick: (labelIdentified: string) => void;
+  setSelectedBoxes: (rows: Row<any>[]) => void;
```

Comment:
Is there a reason why you did not define it like this?
`setSelectedBoxes: (rows: BoxRow[]) => void`